contrib/bash-completion

```diff
@@ -23,9 +23,17 @@ _netctl() {
         COMPREPLY=( $(compgen -W "--help --version list store restore stop-all start stop restart switch-to is-active status enable disable reenable is-enabled edit wait-online" -- "$cur") )
       ;;
       2)
-        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(start|stop|restart|switch-to|is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(is-active|status|enable|disable|reenable|is-enabled|edit|wait-online) ]] &&
           compopt -o filenames &&
           mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(_netctl_profiles)" -- "$cur")
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(stop|restart) ]] &&
+          compopt -o filenames &&
+	  # Suggest only active profiles for stopping/restarting
+          mapfile -t COMPREPLY < <(IFS=$'\n'; compgen -W "$(netctl list | grep '\*' | cut -f2 -d' ')" -- "$cur")
+        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(switch-to|start) ]] &&
```

Comment:
Why change the order? Isn't `@(start|switch-to)` more in line with the rest of the code?

Reply:
I don't know, it's a set of values, order doesn't really matter here. If you think it looks better - OK.
```suggestion
        [[ ${COMP_WORDS[COMP_CWORD-1]} = @(start|switch-to) ]] &&
```